src/nvim/api/buffer.c

```diff
@@ -636,6 +637,185 @@ ArrayOf(Integer, 2) buffer_get_mark(Buffer buffer, String name, Error *err)
   return rv;
 }
 
+/// Return a tuple (row,col) representing the position of the named mark
+///
+/// @param buffer The buffer handle
+/// @param name The mark's name
+/// @param[out] err Details of an error that may have occurred
+/// @return The (row, col) tuple
+ArrayOf(Integer, 2) mark_index(Buffer buffer, String name, Error *err)
```

Comment:
Please prefix api functions in `buffer.c` with `buffer_` (Or, as I expect #4934 to be merged before this,  you can just use `nvim_buf_` upfront, but then you will need to write tests with `helpers.request` for the moment until #4934 gets merged)